### PR TITLE
feat(component): create card component

### PR DESCRIPTION
**Pull request by boiarchukk**
Create new "Card" component 
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/card/tokens/lm-card-token.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+/* eslint-disable no-undef */
+
+const colors = require("@momentum-ui/tokens/dist/colors.json");
+
+const card = {
+  prefix: "lm",
+  component: "card",
+  color: {
+    light: colors.gray[70].name,
+    dark: colors.gray[40].name,
```
Comment (futurethang) [inline web-components/src/components/card/tokens/lm-card-token.js:11]: redundant, already available as `--md-secondary-text-color`

File: web-components/src/components/card/tokens/lm-card-token.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    title: {
+      light: colors.gray[100].name,
+      dark: colors.gray["05"].name
```
Comment (futurethang) [inline web-components/src/components/card/tokens/lm-card-token.js:14]: redundant, already available as `--md-primary-text-color`

File: web-components/src/components/card/tokens/lm-card-token.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  },
+  bg: {
+    light: colors.white[100].name,
+    dark: colors.gray[90].name
```
Comment (futurethang) [inline web-components/src/components/card/tokens/lm-card-token.js:19]: available as `--md-tertiary-white-bg-color`

File: web-components/src/components/card/tokens/lm-card-token.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  border: {
+    light: colors.gray[20].name,
+    dark: colors.gray[80].name
+  }
```
Comment (futurethang) [inline web-components/src/components/card/tokens/lm-card-token.js:24]: this is arguable, we do have a global `--md-quaternary-bg-color` that has these hues, but it says `bg` color not `border` color, so semantically the custom token is more correct. Probably OK to leave this one as is.

File: web-components/src/components/card/scss/settings.scss [hunk 5]
```diff
@@ -0,0 +1,8 @@
+$md-card-width: rem-calc(424);
+$md-card-color: var(--card-color, $md-card-color-light);
```
Comment (futurethang) [inline web-components/src/components/card/scss/settings.scss:2]: use `--md-secondary-text-color` instead

File: web-components/src/components/card/scss/settings.scss [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+$md-card-title-color: var(--card-color-title, $md-card-color-title-light);
```
Comment (futurethang) [inline web-components/src/components/card/scss/settings.scss:3]: use `--md-primary-text-color` instead

File: web-components/src/components/card/scss/settings.scss [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+$md-card-bg: var(--card-bg, $md-card-bg-light);
```
Comment (futurethang) [inline web-components/src/components/card/scss/settings.scss:4]: use `--md-tertiary-white-bg-color`

File: web-components/src/components/card/tokens/md-card-token.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+const card = {
+  prefix: "md",
+  component: "card",
+  color: {
+    light: colors.black[100].name,
+    dark: colors.white[100].name,
+    title: {
+      light: colors.gray[100].name,
+      dark: colors.gray["05"].name
```
Comment (futurethang) [inline web-components/src/components/card/tokens/md-card-token.js:14]: `--md-primary-text-color`

File: web-components/src/components/card/tokens/md-card-token.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  },
+  bg: {
+    light: colors.white[100].name,
+    dark: colors.gray[90].name
```
Comment (futurethang) [inline web-components/src/components/card/tokens/md-card-token.js:19]: `--md-tertiary-white-bg-color`

File: web-components/src/components/card/tokens/md-card-token.js [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  border: {
+    light: colors.gray[20].name,
+    dark: colors.gray[80].name
```
Comment (futurethang) [inline web-components/src/components/card/tokens/md-card-token.js:23]: technically a duplicate of `--md-menu-overlay-border-color`

**Review by futurethang — CHANGES_REQUESTED** [review]
Just auditing some color tokens usages, there are globals available that can do the job in place of most custom tokens.

File: web-components/src/components/card/tokens/md-card-token.js (hunk 10 above)
Comment (marinasmah) [reply web-components/src/components/card/tokens/md-card-token.js]: @futurethang fixed in all mentioned cases

**Review by yananym — APPROVED** [review]

**Pull request merged**